src/index.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import { Provider } from 'react-redux';
+import App from './components/App';
+import store from './store';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Provider store={store}>
+      <App />
+    </Provider>
```

Comment:
After wrapping the `App` in `Provider` it is cleaner to assign the whole thing into a variable e.g. 
```javascript
const wrappedApp = (
  <Provider store={store}>
    <App />
  </Provider>
);
```
What do you think? :thinking:

Reply:
Haahaha!.. It is quite readable but at the expense of memory.